src/popup/App.vue

```diff
@@ -3,7 +3,8 @@
     id="app"
     :class="{
       'show-header': showStatusAndHeader,
-      'display-as-mobile-app': displayAsMobileApp,
```

Comment:
Computed property `displayAsMobileApp` can be removed, because it is no longer used in that file.

Reply:
Done

Reply:
It is not, yet.

Reply:
Ups, yes - you're right. Now I removed it.